nominal/core/run.py

```diff
@@ -35,6 +35,8 @@ class Run(HasRid):
     start: IntegralNanosecondsUTC
     end: IntegralNanosecondsUTC | None
     run_number: int
+    asset: str | None
+    """The RID of the asset representing this run."""
```

Comment:
note: not sure if this field should be `str | None` or just `str`. Supposedly every run should have this field, but in the underlying generated API this field is `str | None`